parlai/agents/transformer/modules.py

```diff
@@ -192,7 +128,7 @@
         self.reduction_type = opt.get('reduction_type', 'mean')
         self.n_segments = opt.get('n_segments', 0)
 
-        self.context_encoder = _build_encoder(
+        self.context_encoder = TransformerGeneratorModel.build_encoder(
```

Comment:
This should be self. Or maybe class(self).

Reply:
Okay, yeah, I was on the fence about whether to add a separate `TransformerMemNetModel.build_encoder()` class method. I've just added one. This adds repetition of code, but it eliminates the strange calling of another class's class method